docs/docs/configuration.md

```diff
@@ -53,6 +53,7 @@ sidebar_position: 3
 | neighbors | []string |  | required,ip | List of neighbor IPs |
 | prepends | int | 0 |  | Number of times to prepend local AS on export |
 | local-pref | int | 100 |  | BGP local preference |
+| default-bgp-local-pref | int | | | BGP default value for local preference|
```

Comment:
Use `go generate` to generate this. The only change would be a space before the `|` character, but good practice to use the code generator here.